--- asdcplib-rel_2_13_1/m4/ax_lib_expat.m4.orig	2025-07-11 01:17:19.000000000 +0200
+++ asdcplib-rel_2_13_1/m4/ax_lib_expat.m4	2025-10-04 08:45:42.900626504 +0200
@@ -109,7 +109,7 @@ AC_DEFUN([AX_LIB_EXPAT],
 
     if test -n "$expat_prefix"; then
         expat_include_dir="$expat_prefix/include"
-        expat_ld_flags="-L$expat_prefix/lib"
+        expat_ld_flags=""
         expat_lib_flags="-lexpat"
         run_expat_test="yes"
     elif test "$expat_requested" = "yes"; then
--- asdcplib-1.12.60/m4/ax_lib_openssl.m4.orig	2009-11-07 01:11:58.000000000 +0100
+++ asdcplib-1.12.60/m4/ax_lib_openssl.m4	2015-04-04 17:46:23.155934920 +0200
@@ -113,7 +113,7 @@
 
     if test -n "$openssl_prefix"; then
         openssl_include_dir="$openssl_prefix/include"
-        openssl_ldflags="-L$openssl_prefix/lib64 -L$openssl_prefix/lib"
+        openssl_ldflags=""
         run_openssl_test="yes"
     elif test "$openssl_requested" = "yes"; then
         if test -n "$openssl_include_dir" -a -n "$openssl_lib_flags"; then
--- asdcplib-rel_2_13_1/m4/ax_lib_xerces.m4.orig	2025-07-11 01:17:19.000000000 +0200
+++ asdcplib-rel_2_13_1/m4/ax_lib_xerces.m4	2025-10-04 09:07:43.283473376 +0200
@@ -110,7 +110,7 @@ AC_DEFUN([AX_LIB_XERCES],
     if test -n "$xerces_prefix"; then
         xerces_include_dir="$xerces_prefix/include"
         xerces_include_dir2="$xerces_prefix/include/xercesc"
-        xerces_ldflags="-L$xerces_prefix/lib"
+        xerces_ldflags=""
         run_xerces_test="yes"
     elif test "$xerces_requested" = "yes"; then
         if test -n "$xerces_include_dir" -a -n "$xerces_lib_flags"; then
